templates/comment.liquid

```diff
@@ -0,0 +1 @@
+{% include "post", post: post %}
```

Comment:
I actually think we can include just the comment snippet here.

Reply:
Done.